.buildkite/scripts/make_image.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 set -euo pipefail
+set -x
```

Comment:
Should this still be here?

Reply:
It's handy when things go wrong, but my plan is to remove it and rebase